godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Transform.kt

```diff
@@ -280,9 +280,9 @@ class Transform(
      */
     fun xform(vector: Vector3): Vector3 =
         Vector3(
-            _basis.x.dot(vector) + _origin.x,
-            _basis.y.dot(vector) + _origin.y,
-            _basis.z.dot(vector) + _origin.z
+            _basis._x.dot(vector) + _origin.x,
```

Comment:
`x`, `y`, and `z` are only visible in GDScript - so therefore they shouldn't be used internally.